resources/home/dnanexus/run_workflows/tests/test_manage_dict.py

```
@@ -1168,71 +1142,87 @@ def test_array_input_with_single_dict_given(self):
         )
 
         input_type = type(
-            output['stage-G9Z2B8841bQY907z1ygq7K9x.somalier_extract_file']
+            output["stage-G9Z2B8841bQY907z1ygq7K9x.somalier_extract_file"]
         )
 
-        assert input_type == list, (
-            'array:file input not converted to a list as expected'
-        )
+        assert (
+            input_type == list
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)